VocaDbModel/DataContracts/Venues/VenueForApiContract.cs

```diff
@@ -20,6 +20,9 @@ public class VenueForApiContract : IEntryWithStatus {
 		[DataMember(EmitDefaultValue = false)]
 		public string AdditionalNames { get; set; }
 
+		[DataMember]
+		public string Address { get; set; }

```

Comment:
Better add comments here. Something like, "Venue address, without country, for example 〒261-8550 Chiba, Mihama Ward, Nakase, 2 Chome−1"